lib/ReactViews/Map/Panels/SharePanel/SharePanel.jsx

```diff
@@ -397,8 +398,37 @@ const SharePanel = observer(
                 )
               }
             />
+            {/* Following code block dependent on existence of "bookmarkHelp" Help Menu Item */}
+            <If
+              condition={this.props.terria.configParameters.helpContent.some(
+                e => e.itemName === "bookmarkHelp"
+              )}
+            >
+              <Text
+                medium
+                textLight
+                isLink
+                onClick={evt => {
+                  evt.preventDefault();
+                  evt.stopPropagation();
+                  viewState.setRetainSharePanel(true);
+                  viewState.showHelpPanel();
+                  viewState.selectHelpMenuItem("bookmarkHelp");
+                }}
```

Comment:
Makes sense to extract it out into new function.

Reply:
Resolved, extracted out to an action on ViewState